app/models/ranking_list.rb

```diff
@@ -0,0 +1,31 @@
+# Contains logic for calculating an array containing ranking entries out of an array of teams and matches.
+class RankingList
```

Comment:
The name will clash a little bit with the rankinglist model we already have. Maybe something shorter like just Ranking is also precise enough? There should definitely be a renaming here in my opinion.

Reply:
Ranking would be fine from my point of view, @bakoe @jonasnoki what do you say?

Reply:
I agree: `Ranking` sounds reasonable. 👍

Reply:
If you change the name of this class, make sure to change the explanatory comment from `ranking_entry.rb`'s first line as well.